src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (element) {
+    for (const key of Object.keys(data)) {
+      const newRow = document.createElement('li');
+
+      newRow.textContent = key;
+      newRow.append(createTree(null, data[key]));
```

Comment:
The recursive call to `createTree` should pass a valid DOM element as the first argument instead of `null`. This is necessary to ensure that the nested structure is correctly appended to the DOM.